src/excmds.ts

```diff
@@ -4251,14 +4251,34 @@ export async function bind(...args: string[]) {
             await browser.commands.update({ name: command.name, shortcut: minimalKeyToMozMap(mapstrToKeyseq(args_obj.key)[0]) })
             await commandsHelper.updateListener()
         }
-        p = config.set(args_obj.configName, args_obj.key, args_obj.excmd)
+        p = config.set(args_obj.configName, args_obj.key, args_obj.excmd).then(() => fillcmdline_nofocus(`${args_obj.key} is bound to ${args_obj.excmd}` + ` in ${args_obj.configName}`))
     } else if (args_obj.key.length) {
         // Display the existing bind
         p = fillcmdline_notrail("bind", args_obj.key, config.getDynamic(args_obj.configName, args_obj.key))
     }
     return p
 }
 
+/**
+     Register key combination from keyboard until enter is pressed, and send it to [[bind]]
+     with provided arguments.
+
+     If you have alwaysqwerty enabled, it will bind commands to physical keys.
+
+     Used same as bind, but without key provided:
+
+         - `bindkey [command]`, then press the keys you want to bind, then hit Enter.
+         - `bindkey --mode=[mode] [command]` also works.
+
+     You can execute it without arguments to see what is bound to the keys you type.
+*/
```

Comment:
Well, currently it does not prefill but executes bind right after you hit enter. And this is why i added output to bind, since otherwise you can't tell what keys have you bound. Do you think it's better to prefill?